package.json

```
@@ -22,6 +22,8 @@
     "index.js"
   ],
   "scripts": {
+    "languages": "node scripts/languages",
+    "prepare": "npm run languages",
```

```suggestion
    "prepare": "node scripts/languages",
```

As there's only one `prepare` action, there's no need for a separate `language` invocation for it.